server/src/main/java/org/apache/cassandra/sidecar/restore/RestoreJobProgress.java

```diff
@@ -106,6 +108,22 @@ private String buildMessage(RestoreJobStatus jobStatus)
         return message + " Current job status: " + jobStatus;
     }
 
+    @VisibleForTesting
+    public List<RestoreRange> allRanges()
```

Comment:
it doesn't need to be public, we only consume it for testing purposes

```suggestion
    List<RestoreRange> allRanges()
```